tools/muscle/compile.sh

```diff
@@ -0,0 +1,9 @@
+set -vexu
+export EM_FLAGS=" -s USE_PTHREADS=0 $EM_FLAGS " ## disable phread
```

Comment:
Can you delete this line, and instead modify the Makefile to replace things like ` -pthread -lpthread` with `-s USE_PTHREADS=0`?

Reply:
fixed